docs/release.sh

```diff
@@ -78,6 +78,11 @@ upload_current_documentation() {
 	src=site/
 	dst=s3://$BUCKET$1
 
+	cache="max-age=3600"
+	if [ "$NOCACHE" ]; then
+		cache="no-cache"
```

Comment:
it actually needs them tho in the shelling out later, I tried without and it doesn't set properly in aws script them :D